caramel/scripts/autosign.py

```diff
@@ -97,11 +101,15 @@ def error_out(message, closer):
 
 def main():
     """Main, as called from the script instance by pyramid"""
-    logging.basicConfig()
-    logger.setLevel(logging.DEBUG)
     args = cmdline()
-    env = bootstrap(args.inifile)
+    config_uri = args.inifile
+
+    env = bootstrap(config_uri)
     settings, closer = env["registry"].settings, env["closer"]
+
+    setup_logging(config_uri)
```

Comment:
This changes the logic. Logging was set up before bootstrapping the application.